protocols/pgp/eth/gtyUs+/tb/PgpEthCaui4GtyTb_behav.wcfg

```diff
@@ -0,0 +1,478 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did this file get included by accident?

Reply:
Yeah, that was on accident. I'll get rid of it